wagtailio/static/css/components/cookie-message.scss

```diff
@@ -0,0 +1,71 @@
+.cookie {
+    $root: &;
+    padding: 0px 10px 10px 10px;
+    position: fixed;
+    right: auto;
+    bottom: 0;
+    left: auto;
+    z-index: 13;
+    display: none; // Doesn't display unless there isn't a cookie
+    width: 100%;
+    pointer-events: none;
```

Comment:
I'm not sure there's a need for removing pointer events if the element is already hidden with `display: none;`